CMakeLists.txt

```diff
@@ -7,10 +7,10 @@ add_subdirectory(lib)
 add_subdirectory(include)
 add_subdirectory(src)
 
-add_library(digiroyale-sim ${DIGIROYALE_SIM_HEADERS} ${DIGIROYALE_SIM_SRC})
+add_library(digiroyale-sim ${DIGIROYALE_SIM_HEADERS} ${DIGIROYALE_SIM_SRC}   "include/digiroyale/simulation/Inventory.hpp" "include/digiroyale/simulation/Survivor.hpp" "src/simulation/Survivor.cpp")
 
-add_executable(digiroyale-bot ${DIGIROYALE_BOT_HEADERS} ${DIGIROYALE_BOT_SRC})
+add_executable(digiroyale-bot ${DIGIROYALE_BOT_HEADERS} ${DIGIROYALE_BOT_SRC}   "include/digiroyale/simulation/Inventory.hpp" "include/digiroyale/simulation/Survivor.hpp" "src/simulation/Survivor.cpp")
 target_link_libraries(digiroyale-bot digiroyale-sim sleepy-discord)
 
-add_executable(digiroyale-test ${DIGIROYALE_TEST_SRC})
+add_executable(digiroyale-test ${DIGIROYALE_TEST_SRC}   "include/digiroyale/simulation/Inventory.hpp" "include/digiroyale/simulation/Survivor.hpp" "src/simulation/Survivor.cpp")
```

Comment:
These need to be reverted, the cmake is set up so that it finds all the source files automatically. Make sure you set up your IDE so that it doesn't automatically add new files to the cmakelists.